From 69b1516c371c2daf74c17fdea1066e78824c48aa Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 15 Mar 2012 14:53:34 +0000
Subject: [PATCH 0178/1000] Mantis 22213: Build: remove exhausted components do
 not link libraries to libraries add needed libraries to LIB_-Variable in
 makefile.common.mak

git-svn-id: svn://localhost/SelfServiceCommon/trunk@307 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/cpp/MassaiCorba/dll/makefile.mak    | 4 +++-
 SelfServiceCommon/Massai/cpp/MassaiCorba/static/makefile.mak | 3 ++-
 2 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/MassaiCorba/dll/makefile.mak b/SelfServiceCommon/Massai/cpp/MassaiCorba/dll/makefile.mak
index b1c92909..8d71b656 100644
--- a/SelfServiceCommon/Massai/cpp/MassaiCorba/dll/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/MassaiCorba/dll/makefile.mak
@@ -26,6 +26,8 @@ MY_LIB1_OBJS = \
     $(_OBJ)\corbasrvr.obj \
     $(_OBJ)\iorparser.obj \
     $(_OBJ)\Realtime.obj \
-    $(_OBJ)\trace.obj
+    $(_OBJ)\trace.obj \
+    $(_OBJ)\corbapoll.obj
+
 
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
diff --git a/SelfServiceCommon/Massai/cpp/MassaiCorba/static/makefile.mak b/SelfServiceCommon/Massai/cpp/MassaiCorba/static/makefile.mak
index 22061c52..6603a3d3 100644
--- a/SelfServiceCommon/Massai/cpp/MassaiCorba/static/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/MassaiCorba/static/makefile.mak
@@ -28,6 +28,7 @@ MY_LIB1_OBJS = \
     $(_OBJ)\corbasrvr.obj \
     $(_OBJ)\iorparser.obj \
     $(_OBJ)\Realtime.obj \
-    $(_OBJ)\trace.obj
+    $(_OBJ)\trace.obj \
+    $(_OBJ)\corbapoll.obj
 
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
-- 
2.41.0.windows.1

